server/routes/users.js

```diff
@@ -3,19 +3,18 @@ const router = express.Router()
 
 const { User } = require( '../database' )
 
-router.post( '/sign-up', (request, response) => {
+router.post( '/sign-up', ( request, response ) => {
   const { username, email, password, lat, long } = request.body
 
-  // TODO: If user exists, return
   User.signUp({ username, email, password, lat, long })
-    .then( user => response.status(201).json( user ))
-    .catch( error => response.status(500).json({ error: error.message  }))
+    .then( user => response.status( 201 ).json( user ))
+    .catch( error => response.status( 500 ).json({ error: "The email or username you entered were already taken." }))
 })
 
-router.post( '/login', (request, response) => {
+router.post( '/login', ( request, response ) => {
   User.login( request.body )
-    .then( user => response.cookie( 'user_id', user.id ).status(200).json( user ))
-    .catch( error => response.status(500).json({ error: error.message }))
+    .then( user => response.cookie( 'user_id', user.id ).status( 200 ).json( user ))
+    .catch( error => response.status( 500 ).json({ error: error.message }))
```

Comment:
why are you using `error.message` here instead of a custom error message here?

Reply:
The `User.login` method throws an error which I catch here in my route.